apps/docs/src/pages/en/introduction.md

```diff
@@ -34,24 +34,25 @@ However we also see that Astro has given us a good baseline framework to potenti
 
 The challenge right now is that there is no opinionated way to "resume" what your components did from the server to the browser. So I thought: What if there is a library of components and utilities to help us do this?
 
-In this project, our motivation is to get to this future. This is our *next frontier 🚀*
+In this project, our motivation is to get to this future. This is our _next frontier 🚀_
 
 ## Some Tradeoffs
 
 The Astro framework is very clear and focused on what they provide: top-tier performance for content-focused websites. In doing so, it has introduced a new world of architecture to a lot of developers.
 
 Once we take off in pursuit of our project goals, we are aware of some tradeoffs that will be made.
- 
+
 As we build, we aim to minimize the performance hit to your applications.
 
 We will also make this clear in our library documentation as we see them.
 
 ## Packages
 
-| Package | Release notes | Description |
-| --- | --- | --- |
-| [form](/en/api/form/form-component) | [![npm](https://img.shields.io/npm/v/@astro-reactive/form)](./packages/form/RELEASE.md) | a dynamic form which can be modified programmatically |
-| [validator](https://github.com/ayoayco/astro-reactive-library/blob/main/packages/validator/README.md) | [![npm](https://img.shields.io/npm/v/@astro-reactive/validator)](./packages/validator/RELEASE.md) | validators for editable fields |
-| data-grid | 🛠 | a dynamic data grid of values |
-| themes | 🛠 | easy-to-use, accessible, consistent cross-browser styles |
-| viz | 🛠 | data visualization that emits and responds to events |
+| Package                                                                                              | Release notes                                                                                     | Description                                              |
+| ---------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| [form](/en/api/form/form-component)                                                                  | [![npm](https://img.shields.io/npm/v/@astro-reactive/form)](./packages/form/RELEASE.md)           | a dynamic form which can be modified programmatically    |
```

Comment:
![image](https://user-images.githubusercontent.com/87543808/210422900-afda615b-33d9-4450-8fd9-b35eb48abc19.png)
I opened this in rich diff mode by clicking on this icon then I clicked the  `form`  package link, it is giving 404 error. Please check it once.
I guess it should be replaced with this `https://github.com/astro-reactive/astro-reactive/blob/main/packages/form/README.md` @ayoayco ?

All other links are working fine!!

Reply:
It should be fine once this is merged. No need to change :)

Reply:
Woops I just noticed this is not the repo README but on our docs site. @Preet-Sojitra you are right.